src/main/java/com/example/gradecalculator/entities/Subject.java

```diff
@@ -21,13 +21,15 @@ public class Subject {
     @JoinColumn(name = "school_year")
     private SchoolYear schoolYear;
 
-    public Subject() {
-    }
-
     public Subject(String name, String description) {
         this.name = name;
         this.description = description;
     }
+
+    public Subject() {
```

Comment:
why was this touched?

Reply:
accidentally

Reply:
if you accidentally changed it, just change it back again so there is no unnecessary changes in the code